scss/tech.scss

```diff
@@ -0,0 +1,43 @@
+.tech {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding: 50px;
+  width: 100%;
+
+  .box {    
+    align-items: center;
+    background-color: $green;
+    border: 2px;
+    border-radius: 5px;
+    box-shadow: -10px -10px 0 0 white, -10px -10px 0 3px $green;
+    display: inline-flex;
```

Comment:
why not only flex?

Reply:
Changed to flex and it remains the same (don't remember why I used inline-flex)